src/commands/deploypackageCommand.ts

```diff
@@ -93,6 +103,80 @@ export class DeploypackageCommand implements ICommand {
           });
         }
       });
+  }
+
+  private handleDeployResponse(response: any) {
+    this.output.appendLine('Deploy package completed');
+    this.output.appendLine(`Status: ${response.status}`);
+    this.output.appendLine('============================\n');
+
+    console.log(response);
+    switch (response.status) {
+      case DeployStatus[DeployStatus.Failed]: // Failed
+      case DeployStatus[DeployStatus.Succeeded]: // Succeeded
+      case DeployStatus[DeployStatus.SucceededPartial]: // Succeeded partially
+        this.displayDeployResponse(response);
+        break;
+      case DeployStatus[DeployStatus.InProgress]: // Other status
+      case DeployStatus[DeployStatus.Canceling]:
+      case DeployStatus[DeployStatus.Pending]:
+        console.log("Should not happend")
+        break;
+    }
+
+    vscode.window.showInformationMessage(`Deployment of package ${response.status}`, { title: 'Show output', action: 'SHOW_OUTPUT' })
```

Comment:
Faudrait changer ça, c'est vraiment weird de voir "Deployement of package success" comme message